docs/known_bugs.rst

```diff
@@ -1,6 +1,14 @@
 Known Issues
 ============
 
+On some platforms, install fails due to `scikit-image`
```

Comment:
Would it be useful to note that this is only applicable to users working in a conda environment? It seems obvious, but... *shrug*

Reply:
Good call!